src/software/firmware/srcs/respirator.cpp

```diff
@@ -191,6 +195,14 @@ void loop() {
     /********************************************/
     // END OF THE RESPIRATORY CYCLE
     /********************************************/
+
+    // Because this kind of LCD screen is not reliable, we need to reset it every 5 min or so
+    cyclesBeforeScreenReset--;
+    if (cyclesBeforeScreenReset <= 0) {
+        DBG_DO(Serial.println("resetting LCD screen");)
+        resetScreen();
```

Comment:
Any idea how much time this call can take? Is there a  risk that the pause between two cycles is too long?

Reply:
:+1: